src/scene/vcPOI.cpp

```diff
@@ -43,9 +43,7 @@ class vcPOIState_General
 public:
   vcPOIState_General(vcPOI *pParent)
     : m_pParent(pParent)
-  {
-
-  }
+  {}
```

Comment:
This seems like an unnecessary change? Is this consistent with other constructors now?